client/elife-theme/src/elements/TextField.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+/**
+ * TODO
+ * this should be removed once it's implemented in pubsweet
+ * right now they don't have colorWarning
+ */
+const borderColor = ({ theme, validationStatus = 'default' }) =>
+  ({
+    error: theme.colorError,
+    success: theme.colorSuccess,
+    warning: theme.colorWarning,
+    default: theme.colorBorder,
+  }[validationStatus])
+
+/**
+ * TODO
+ *
+ * font-size & line-height might change to fontSizeBase in the future
+ *
+ * for some reason th didn't work for spacing
+ * padding: ${th('space.2') + 'px'};
```

Comment:
Suggestions welcomed

Reply:
See #249

Reply:
This still doesn't answer my question. Why is this ```padding: ${props => `${props.theme.space[2]}px`}; ``` working and `padding: ${th('space.2') + 'px'};` is not? My guess is that it's some tiny stupid detail difference that I can't see right now.

Reply:
Because `th('space.2')` returns a function, not a number, so you can't add a string to it.

Reply:
That makes sense. I guess doing something like `props => th('space.2')(props) + 'px'` would work but that defeats the purpose of `th`.